package.json

```diff
@@ -14,6 +14,7 @@
     "@types/react-dom": "16.9.1",
     "compression-webpack-plugin": "^3.0.0",
     "css-loader": "^3.2.0",
+    "file-loader": "^5.0.2",
```

Comment:
You never import any images in your TS so you don't need this plugin, it'll just slow down the compiler